variables.yml

```diff
@@ -170,8 +171,9 @@ networks:
       stable_version: 11.3.9
     tx_weight_to_gas_ratio: 25000
   moonriver:
-    rpc_url: https://rpc.api.moonriver.moonbeam.network
-    wss_url: wss://wss.api.moonriver.moonbeam.network
+    public_rpc_url: https://rpc.api.moonriver.moonbeam.network # for wallets
+    rpc_url: RPC-API-ENDPOINT-HERE
```

Comment:
I'm not very fond of this approach. There are lots of tables where it does not make a lot of sense. For example: 

![image](https://user-images.githubusercontent.com/64150856/156355188-9eb4b44a-7aa1-4fb6-926f-3ce0aad26965.png)

Not sure what is the best approach, probably a link to the Network Endpoint page?

Reply:
I have a link to the Network Endpoint page everywhere that the rpc url is, I can remove it from here?

Reply:
Do you want me to remove the variable and manually put RPC-API-ENDPOINT-HERE only in some places?